richtextfx/src/integrationTest/java/org/fxmisc/richtext/style/StylingTests.java

```diff
@@ -0,0 +1,160 @@
+package org.fxmisc.richtext.style;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.fxmisc.flowless.Cell;
+import org.fxmisc.flowless.VirtualFlow;
+import org.fxmisc.richtext.InlineCssTextAreaAppTest;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.nitorcreations.junit.runners.NestedRunner;
+
+import javafx.scene.Node;
+import javafx.scene.layout.Region;
+import javafx.scene.shape.Path;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+
+@RunWith(NestedRunner.class)
+public class StylingTests extends InlineCssTextAreaAppTest {
+
+    private final static String firstWord = "Hello ";
+    private final static String styledWord1 = "World";
+    private final static String moreText = " and also the ";
+    private final static String styledWord2 = "Sun";
+    private final static String remainingLine = " and Moon";
```

Comment:
Relating to line 89's comment, since the string itself does not contain the style, it should equal to what the text it is assigned is. Otherwise, `styledWord1` is unstyled in one moment and styled in a later moment.